sdk/php/runtime/main.go

```diff
@@ -25,11 +25,10 @@ type PhpSdk struct {
 func New(
 	// Directory with the PHP SDK source code.
 	// +optional
+	// +defaultPath="/sdk/php"
 	sdkSourceDir *dagger.Directory,
 ) *PhpSdk {
 	if sdkSourceDir == nil {
-		// TODO: Replace with a *default path from context* when
-		// https://github.com/dagger/dagger/pull/7744 becomes available.
```

Comment:
`if sdkSourceDir == nil` should actually return an error now, since when used as `--sdk=php` it should be provided by `defaultPath`.

Reply:
I think we can eliminate `sdkSourceDir == nil` now since it is already available via the defaultPath.

Reply:
I kept it in Python but just return an error. That's because the argument is a pointer and I don't to hit any situation where it can be `nil` which would raise a panic.

Reply:
I see. Fixed on https://github.com/dagger/dagger/pull/8369/commits/ada61c18667d24329ace94bc94be148ba2054201